packages/jaeger-ui/src/components/SearchTracePage/SearchResults/ResultItem.js

```diff
@@ -60,55 +60,57 @@ export default class ResultItem extends React.PureComponent<Props> {
-                {numSpans} Span{numSpans > 1 && 's'}
-              </Tag>
-              {Boolean(numErredSpans) && (
-                <Tag className="ub-m1" color="red">
-                  {numErredSpans} Error{numErredSpans > 1 && 's'}
+      <div className="ub-pt3">
+        <div className="ResultItem">
+          <ResultItemTitle
+            duration={duration}
+            durationPercent={durationPercent}
+            isInDiffCohort={isInDiffCohort}
+            linkTo={linkTo}
+            toggleComparison={toggleComparison}
+            traceID={traceID}
+            traceName={traceName}
+            disableComparision={disableComparision}
+          />
+          <Link to={linkTo}>
+            <Row>
+              <Col span={4} className="ub-p2">
+                <Tag className="ub-m1" data-test={markers.NUM_SPANS}>
+                  {numSpans} Span{numSpans > 1 && 's'}
                 </Tag>
-              )}
-            </Col>
-            <Col span={16} className="ub-p2">
-              <ul className="ub-list-reset" data-test={markers.SERVICE_TAGS}>
-                {sortBy(services, s => s.name).map(service => {
-                  const { name, numberOfSpans: count } = service;
-                  return (
-                    <li key={name} className="ub-inline-block ub-m1">
-                      <Tag
-                        className="ResultItem--serviceTag"
-                        style={{ borderLeftColor: colorGenerator.getColorByKey(name) }}
-                      >
-                        {name} ({count})
-                      </Tag>
-                    </li>
-                  );
-                })}
-              </ul>
-            </Col>
-            <Col span={4} className="ub-p3 ub-tx-right-align">
-              {formatRelativeDate(startTime / 1000)}
-              <Divider type="vertical" />
-              {timeStr.slice(0, -3)}&nbsp;{timeStr.slice(-2)}
-              <br />
-              <small>{fromNow}</small>
-            </Col>
-          </Row>
-        </Link>
+                {Boolean(numErredSpans) && (
+                  <Tag className="ub-m1" color="red">
+                    {numErredSpans} Error{numErredSpans > 1 && 's'}
+                  </Tag>
+                )}
+              </Col>
+              <Col span={16} className="ub-p2">
+                <ul className="ub-list-reset" data-test={markers.SERVICE_TAGS}>
+                  {sortBy(services, s => s.name).map(service => {
+                    const { name, numberOfSpans: count } = service;
+                    return (
+                      <li key={name} className="ub-inline-block ub-m1">
+                        <Tag
+                          className="ResultItem--serviceTag"
+                          style={{ borderLeftColor: colorGenerator.getColorByKey(name) }}
+                        >
+                          {name} ({count})
+                        </Tag>
+                      </li>
```

Comment:
For this, the `<Tag...>` seems pretty heavy. What about using something home-grown?

```jsx
<li
  key={name}
  className="ResultItem--serviceTag"
  style={{ borderLeftColor: colorGenerator.getColorByKey(name) }}
>
  {name} <span className="ResultItem--serviceCount">{count}</span>
</li>
```

```css
.ResultItem--serviceTag {
  background: #fff;
  border-left: 15px solid transparent;
  border-radius: 4px;
  box-shadow: 0 0 1px rgba(0, 0, 0, 0.5);
  color: rgba(0, 0, 0, 0.65);
  display: inline-block;
  font-size: 0.9em;
  line-height: 1;
  margin: 0.3em 0.35em;
  padding: 0.3em;
}

.ResultItem--serviceCount {
  font-size: 0.9em;
  opacity: 0.5;
  padding-left: 0.2em;
}
```